doc/Language/variables.pod6

```diff
@@ -491,7 +492,7 @@ new-location; # OUTPUT: «outside␤»
 =end code
 
 To make C<new-location()> print C<nowhere>, make C<$location> a dynamic
-variable using L<the * twigil|#The_*_Twigil>.
+variable using L<the * twigil|#The_*_Twigil>. This twigil makes the compiler look up the symbol in the calling scope instead of the outer scope which is the case otherwise.
```

Comment:
The compiler always looks in the current scope first. Looking into outer/caller's scopes is always a fallback. This description makes it sound as if only the caller's scope is searched.